_join/open-positions/front-end-design-intern.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 18F – Front end design intern
+permalink: /join/front-end-design-intern/
+layout: primary
+lead: 
+app_close_date: 2017-07-26
+subnav_title: Front end design intern
+subnav_items:
+  - text: Basic information
+    permalink: /#basic-information
+  - text: Role summary
+    permalink: /#role-summary
+  - text: How to apply
+    permalink: /#how-to-apply
+breadcrumb: true
+published: true
+listed: true
+---
+
+## Basic information
+
+**Title:** Service/visual design intern
```

Comment:
Looks like this isn't the right title here, but on the preview it says the right title with an extra `title: 18F -" in front, even though I don't see that here.